archimedes/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "archimedes"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+numpy = "^1.19.5"
+FlowCytometryTools = "^0.5.0"
+pandas = "^1.2.1"
+scikit-learn = "^0.24.1"
+scipy = "^1.6.0"
+rpy2 = "^3.4.2"
+pylint = "^2.6.0"
+black = "^20.8b1"
+docker = "^4.4.1"
+dataclasses-json = "^0.5.2"
+
+[tool.poetry.dev-dependencies]
+pytest = "^5.2"
+
+[tool.poetry.scripts]
+archimedes-checker = 'archimedes.checker:main'
```

Comment:
Allows DSL script checker and the runner to be executed as scripts on their own.